BatchProfiler/bputilities.py

```diff
@@ -420,7 +420,7 @@ def build_cellprofiler(
     touch_job = "CellProfiler-touch-%s" % git_hash
```

Comment:
The BatchProfiler directory should be removed from the CellProfiler repo at this point (didn't I do that?). The BatchProfiler repo has all the changes. So no worries here, but BP will need to be changed to do different things to build depending on the version. I will fix when this goes in.